src/components/Result.html

```diff
@@ -21,7 +21,7 @@
 						class="quiz-container flex-row-center padding-small-card shade"
 					>
 
-          <img class="quiz-result-img imgTransition" src="../components/img/trophy3.jpg" alt="Result">
+          <img class="quiz-result-img imgTransition" src="../components/img/trophy3.jpg" alt="">
 
```

Comment:
suggestion - add some alt text about the contents of the image

Reply:
Okay, I'll do that.

Reply:
Okay, I'll do that.